functions/ieeg_getLabelXyzDestrieux.m

```diff
@@ -0,0 +1,201 @@
+%
+%   Returns Destrieux labels to input xyz coordinates based on volume labelling
+%   Requires vistasoft
+%
+%   labs = ieeg_getLabelXyzDestrieux(xyzs, FSdir)
+%   [labs, labs_val] = ieeg_getLabelXyzDestrieux(xyzs, FSdir, hemi, rad)
+%       
+%       xyzs =              nx3 array of n [x, y, z] coordinates
+%       FSdir =             path to FreeSurfer directory for the current subject; e.g. */derivatives/freesurfer/sub-<subject>
+%       hemi (optional) =   string: 'left', 'L', 'l' or 'right', etc. corresponding to hemisphere of subject. 'left' by default
```

Comment:
hemi is actually only used here to load the Destrieux colortable, which is hemisphere agnostic, but not for any further processing